tools/standard-test/README.md

```diff
@@ -17,13 +17,31 @@ nvm installation script from nvm github repo to help install nvm on the dockerfi
 
   * You must make this edit on the LINUX host and restart docker deamon for OpenSearch process to run.
     This will also prevent errors during Dashboards integtest (cypress) so that headless chrome will not crash.
-    ```
-    # Edit /etc/sysconfig/docker to have nofiles to 65535, and default shared memory to 1024MiB
-    # OPTIONS="--default-ulimit nofile=65535:65535 --default-shm-size 1024000000"
-    # Then restart docker
-    sudo systemctl stop docker
-    sudo systemctl start docker
-    ```
+    * Option 1:
```

Comment:
Can users choose either of the option? Can we add the recommended option?

Reply:
Yes let me add that one.